instrumentation/opentelemetry-instrumentation-django/src/opentelemetry/instrumentation/django/__init__.py

```diff
@@ -32,7 +32,8 @@
 
 Exclude lists
 *************
-To exclude certain URLs from being tracked, set the environment variable ``OTEL_PYTHON_DJANGO_EXCLUDED_URLS`` with comma delimited regexes representing which URLs to exclude.
+To exclude certain URLs from being tracked, set the environment variable ``OTEL_PYTHON_EXCLUDED_URLS`` or ``OTEL_PYTHON_DJANGO_EXCLUDED_URLS``
+with comma delimited regexes representing which URLs to exclude.
```

Comment:
Maybe mention the precedence when both are set?

Reply:
Would this wording make that clear, or should we go with a different one?
> To exclude certain URLs from being tracked, set the environment variable ``OTEL_PYTHON_{instrumentation}_EXCLUDED_URLS`` (or ``OTEL_PYTHON_EXCLUDED_URLS`` as fallback) with comma delimited regexes representing which URLs to exclude.

Reply:
@lonewolf3739 please resolve if you are satisfied with the change. thanks